src/components/pages/Incidents/Map/Marker.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Marker = props => {
+  return (
+    <div
+      className="marker-container"
+      style={{
+        height: '5px',
+        width: '5px',
+        backgroundColor: 'red',
```

Comment:
Is there a reason for having these styles inline?

Reply:
Just leaving them online for now until the organization for styling is more clear. Will move into a less style sheet soon.

Reply:
Let's get that organization figured out and apply more styling before we ship this feature.